### ISSUE TITLE
Ability to change project entry point

### ISSUE THREAD

**Issue by NuroDev** [denoland/deploy_feedback#445]
### What problem are you trying to solve?

Using Fresh as an example, if you change your project structure, such as moving your source code into a `src/` directory, then any existing deployments will begin to fail as you cannot change / update the entrypoint path unless you re-create the project

### Describe the solution you'd like

A new input / select drop down, similar to the project creation screen, where you can select a new entrypoint for the project.

### Describe alternatives you've considered

In my case the only alternative I had found is to delete the project & re-create it. Which is fine but not ideal.

**Comment by ry** [comment]
We do support this, it's in the settings tab.

**Comment by NuroDev** [comment]
Really? I cannot see it. Unless you are referring to needing to "Unlink" my git integration & re-link it to set it again?

These are all the items / sections I can see in a projects settings page:
- Project name
- Environment variables
- Git Integration
- Domains
- Transfer project
- Delete project

**Comment by vwkd** [comment]
Yes, unlink and relink repo should AFAIR do the trick. Though I agree changing only the entry point could be integrated better than to relink the whole repo.
